rskj-core/src/main/java/co/rsk/config/ConfigLoader.java

```diff
@@ -128,4 +150,52 @@ private Config getNetworkDefaultConfig(Config userConfig) {
         logger.info("Network not set, using mainnet by default");
         return ConfigFactory.load(MAINNET_RESOURCE_PATH);
     }
+
+    private static void verify(String key, @Nullable ConfigValue expectedValue, ConfigValue actualValue) {
+        if (expectedValue == null) {
+            throw unexpectedKeyException(key, actualValue);
+        }
+
+        switch (actualValue.valueType()) {
+            case OBJECT:
+                if (!expectedValue.valueType().equals(ConfigValueType.OBJECT)) {
+                    throw typeMismatchException(key, expectedValue, actualValue);
```

Comment:
instead of throwing, it would be nice to collect them, and throw/log them all at once.

Reply:
done